WordPress/UITests/Tests/NotificationTests.swift

```diff
@@ -46,13 +46,15 @@ class NotificationTests: XCTestCase {
     }
 
     func testLikeNotification() throws {
-        let (updatedNotificationsScreen, previousLikes) = try TabNavComponent()
+        // Get number of likes before liking the notification
+        let (updatedNotificationsScreen, initialLikes) = try TabNavComponent()
             .goToNotificationsScreen()
             .openNotification(withSubstring: .commentNotificationString)
             .getNumberOfLikesForNotification()
 
+        // Tapping like and verify that like count increased
         updatedNotificationsScreen
             .likeComment()
-            .verifyCommentLiked(likes: previousLikes)
+            .verifyCommentLiked(expectedLikes: initialLikes + 1)
```

Comment:
This approach (providing the expected value, instead of incrementing it inside the function) looks more clear 👍